modules/logger/logger.py

```diff
@@ -13,6 +13,8 @@
 # pylint: disable-next=unused-import
 import types
 
+import numpy as np
+from PIL import Image
 from ..read_yaml import read_yaml
```

Comment:
Add a space above this line. We separate imports from our own files from imports of external libraries (btw, we also separate Python default libraries before those).